src/Tag/Columns.php

```diff
@@ -47,7 +47,7 @@ public function open($attr, &$ahtml, &$ihtml)
 				if ($attr['VALIGN'] === 'J') {
 					$valign = 'J';
 				} else {
-					$valign = self::ALIGN[$attr['VALIGN']];
```

Comment:
The attr uses uppercase here. It is possible that the case will mismatch.

Reply:
Noted

Reply:
Wait that doesn't make sense -
 `self::ALIGN[$attr['VALIGN']];` if $attr['VALIGN'] is upper case like `CENTER` then `self::ALIGN['CENTER']` breaks